docs/UserGuide.md

```diff
@@ -506,13 +506,13 @@ If your changes to the data file makes its format invalid, MyInsuRec will discar
 ## 6. FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app, then overwrite the `myinsurec.json` file created by the app with the version of the file from the previous computer.
+**A**: Install the app, then overwrite the **myinsurec.json** file created by the app with the version of the file from the previous computer. The **myinsurec.json** is located in **[JAR file location]/data/myinsurec.json**.
 
 **Q**: I accidentally closed the app, will my data still be there?<br>
 **A**: Yes, your data is saved automatically after every action.
 
 **Q**: My computer does not recognise the file type jar. How do I open the app?<br>
-**A**: Install Java version 11 and above from the official Java website.
+**A**: Check if your computer already has Java 11 or above install from [8.1 Checking your system's Java version](#81-checking-your-systems-java-version). If not, follow the instructions to install Java.
```

Comment:
`installed` instead of install for the first sentence. Just a minor grammar issue